CMakeLists.txt

```diff
@@ -68,15 +68,22 @@ set(OMR_INSTALL_DATA_DIR ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME} CACHE PATH "In
 ###
 
 if (OMR_ENV_DATA64)
-	find_package(Threads)
+	find_package(Threads QUIET)
 	if(Threads_FOUND)
 		# Threads mush be found before we can look for CUDA.
 		# FindCuda will error out if Threads is missing, even though CUDA itself is optional.
```

Comment:
also should correct this comment as it no longer matches what will happen

Reply:
I'm not sure what needs to be corrected here.

Reply:
Once you added "QUIET" it's no longer try that "FindCuda will error out" is it? Or am I reading too much into this comment?

Reply:
Oh I see what you're saying. FindCUDA is already optional, now we're just making it quiet too.  The comment is documenting what I consider a bug in FindCUDA module. FindCUDA treats FindThreads as required, even if FindCUDA itself is optional. So we have to ensure that FindThreads exists before we use FindCUDA.